src/components/pastAnalysis/PastAnalysisTable.tsx

```diff
@@ -25,9 +24,17 @@ interface tableRowsChild {
   cells: { [title: string]: ReactNode }[]
 }
 
+interface TableStates {
```

Comment:
A general question: where are we setting it to grab 10 results at a time to display? Is there a specific place we decide this, or is this default behaviour?

Reply:
That value is specified in `analyseReducer` in `initialIPrevAnalysesState`